app/views/assignment_journals/edit.html.slim

```diff
@@ -0,0 +1,7 @@
+.row
+  .col-xs-12
+    h1= t_title(:edit)
+
+== render 'form'
```

Comment:
Single `=` should be fine for `render` (`==` disables HTML escaping which is dangerous/rarely needed)